src/Codeception/Configuration.php

```diff
@@ -646,7 +646,7 @@ private static function prepareParams($settings)
             }
 
             // .env and ini files
-            if (preg_match('~(\.ini|\.env)$~', $paramStorage)) {
+            if (preg_match('~(\.ini$|\.env)~', $paramStorage)) {
```

Comment:
I see, that's good. I think that's a developer friendly approach 👍

I don't know if this functionality is covered in the testing suite already, as I've only recently started to participate here, but I think adding a test that catch the edge case you're fixing in this PR would prevent future regressions.

Basically just a test that makes sure `.env.something` is loaded correctly :)